t/op/array.t

```diff
@@ -265,21 +265,29 @@ is ($got, '');
 }
 
 sub test_arylen {
-    my $ref = shift;
+    my ($ref, , $fixed, $desc) = @_;
```

Comment:
Minor: the double comma doesn't seem useful and may confuse people looking over the test in the future.